ansible/roles/kong-api/defaults/main.yml

```diff
@@ -763,42 +672,6 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
 
-  - name: createissuer
-    uris: "{{ badge_service_prefix }}/v1/issuer/create"
-    upstream_url: "{{ learning_service_url }}/v1/issuer/create"
-    strip_uri: true
-    plugins:
-    - name: jwt
-    - name: cors
-    - "{{ statsd_pulgin }}"
-    - name: acl
-      config.whitelist:
-        - 'badgeCreate'
-    - name: rate-limiting
-      config.policy: local
-      config.hour: "{{ medium_rate_limit_per_hour }}"
-      config.limit_by: credential
-    - name: request-size-limiting
-      config.allowed_payload_size: "{{ medium_request_size_limit }}"
-
-  - name: createLocation
```

Comment:
Isn't this API used to create location?

Reply:
NO, this is for FCM related location details. 
for master data we are using masterLocationCreate

Reply:
Thanks for the clarification @amit-tarento